### PR TITLE
Handle github remote authentication failures.

### PR DESCRIPTION
**Pull request by alexperovich**

### CODE REVIEW

File: src/Maestro/Maestro.Web/Startup.Auth.cs [hunk 1]
```diff
@@ -62,6 +66,16 @@ private void ConfigureAuthServices(IServiceCollection services)
...
                             },
+                            OnRemoteFailure = async context =>
+                            {
+                                var logger = context.HttpContext.RequestServices.GetRequiredService<ILogger<GitHubAuthenticationHandler>>();
+                                logger.LogError(context.Failure, "Github authentication failed.");
+                                context.HandleResponse();
+                                var res = context.HttpContext.Response;
+                                res.StatusCode = (int)HttpStatusCode.Forbidden;
+                                res.GetTypedHeaders().ContentType = new MediaTypeHeaderValue("text/html");
+                                await context.HttpContext.Response.Body.WriteAsync(Encoding.UTF8.GetBytes("403; Authentication failed. <a href='/Account/SignIn'>Sign In</a> again."));
```
Comment (ChadNedzlek) [inline src/Maestro/Maestro.Web/Startup.Auth.cs]: This line is really long and really indented, so it's hard to read most of it.
It's not a particularly well formed HTML document... Doesn't have a title, doesn't have the metadata, isn't going to look right...  Should we 302 them to somewhere instead, so we don't have to have inline HTML right here? And have a real page here?

Comment (alexperovich) [reply src/Maestro/Maestro.Web/Startup.Auth.cs]: I could use a status code pages thing and redirect them to `/Error`?

Comment (ChadNedzlek) [reply src/Maestro/Maestro.Web/Startup.Auth.cs]: Is that easy in core land? That's probably good.

Comment (alexperovich) [reply src/Maestro/Maestro.Web/Startup.Auth.cs]: Its a few lines of stuff, I'd have to write the Error page though.

**Pull request merged**